src/components/singleShoppingList.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  Checkbox,
+  Dialog,
+  DialogTitle,
+  ListItem,
+  List,
+  ListItemIcon,
+  ListItemText,
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+  root: {
```

Comment:
add a `// TODO` here to make these styles responsive so that they work with different screen sizes. We'll need to create an issue for this as well. We want the web app to be usable on mobile browsers too (eventually)